src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss

```diff
@@ -2,10 +2,54 @@
 
 $toolbarBackgroundColor: $white;
 $shadowColor: $mischka;
+$buttonsContainerBackgroundColor: $alabaster;
+
+$buttonsContainerBorder: 1px $gallery solid;
 
 .toolbar {
     height: 60px;
     margin: 0;
     background: $toolbarBackgroundColor;
     box-shadow: 0 3px 2px $shadowColor;
 }
+
+.buttons {
+    display: inline-block;
+    border-left: $buttonsContainerBorder;
+    border-right: $buttonsContainerBorder;
+    vertical-align: top;
+    background-color: $buttonsContainerBackgroundColor;
+}
+
+.pull-left {
+    float: left;
+}
+
+.pull-right {
+    float: right;
+}
+
+.icons,
+.locale {
+    display: inline-block;
+}
+
+.icons {
+    padding: 0 20px;
+    vertical-align: middle;
+}
+
+.icon {
+    display: inline-block;
+    padding: 0 20px 0 0;
+
+    &:last-child {
+        padding: 0;
+    }
+}
+
+.locale {
+    button {
+        border-left: $buttonsContainerBorder;
```

Comment:
Can't this also be applied to the `.locale` directly? Then it wouldn't bother that much if we change the child from a `button` to something else one day.